lib/ExecutionContext/TraceEvents.cpp

```diff
@@ -53,6 +53,13 @@ void TraceEvent::dumpTraceEvents(
   auto process = processName.empty() ? "glow" : processName;
 
   std::ofstream file(filename);
+
+  // Print an error message if the output stream can't be created.
+  if (!file.good()) {
```

Comment:
should this be is_open()? (http://www.cplusplus.com/reference/fstream/ifstream/is_open/)

I was not able to find a good documentation on good() vs is_open() yet.